site/docs/es/plugins/emoji.md

```diff
@@ -36,7 +36,7 @@ import { EmojiFlavor, emojiParser } from "@grammyjs/emoji";
 
 // Esto se llama Context Flavoring
 // Puedes leer más en:
-// https://grammy.dev/guide/context.md#transformative-context-flavors
+// https://grammy.dev/es/guide/context.html#transformative-context-flavors
```

Comment:
The anchor will not work, the section is named differently in Spanish.

Reply:
Actually I took the url anchor directly from the translated site. You should [check it](https://grammy.dev/es/guide/context.html#transformative-context-flavors) yourself (and on [GitHub](https://github.com/grammyjs/website/blob/main/site/docs/es/guide/context.md?plain=1#L377)).